src/routes/blog/ide-ux-bof-scipy-2024/+page.md

```diff
@@ -0,0 +1,141 @@
+
+All told, there was a surprising amount of interest and discussion around this once-niche topic, signaling that GPU compute has become something much closer to a mainstream need in the scientific space.
+Thus, while the discussion is of course of most interest to GPU companies like Nvidia, it suggests an IDE plugin to support this profiling tooling and other GPU integration features might be of significant benefit to the wider user community.
+
+
+## Spyder feedback
+
+While most of the discussion focused on topics applicable to broader categories of IDEs, there were a few specific bits of Spyder feedback.
+In particular, attendees praised Spyder's plotting functionality, though some expressed a preference for the non-default mode of displaying plots in separate windows over the Plots pane.
+Others mentioned that the one killer feature for them that it was missing was multi-line editing, a common request over the years and something we plan to work on for forthcoming Spyder versions, along with support for collaborative editing.
+
+A particularly salient comment was one attendees request that "We need RStudio for Python", with another replying that Spyder fills most of that gap.
+Perhaps the main RStudio feature not built in is Quarto/RMarkdown, though we are currently exploring the possibility of helping develop a plugin supporting that.
+Other comments, such as a desire for GitHub Copilot, better data exploration and visualization capabilities, and support for connecting to remote JupyterHub instances, are also features we already offer or are developing or exploring.
+
+
+## Next steps
+
+Overall, the broader learnings gained here for IDE and interface developers were perhaps less in the detailed specifics of each comment, but rather the common themes users focused the most on.
+Perhaps the least surprising was interest in generative AI, suggesting that IDE developers should continue to explore integration possibilities, but keeping a critical eye while doing so.
+By contrast, the most surprising was the extensive discussion about GPU compute and how IDEs and tools can help with that, which seems to indicate that IDE plugins for assisting such might be valuable.
+
+Another area of user emphasis was the importance of interactive data exploration for a variety of formats and structures.
+While a subset of such could be built into the IDE (as we have with Spyder's Variable Explorer), given the diversity of scientific data types across fields this suggests the need for a more pluggable/extensible approach to variable exploration.
+Finally, the consensus was unresolved on desktop versus web-based IDEs, though participants did identify a need for a common framework to connect local IDEs to remote platforms like JupyterHub.
+To that end, we've developed an initial version of for Spyder 6.0 as a JupyterHub plugin, with more extensive capabilities to follow in Spyder 6.1, and could eventually be developed into such an common interface.
```

Comment:
```suggestion
To that end, we're working to provide those facilities in Spyder 6.1, and could eventually be developed into such an common interface.
```

- In 6.0 we have our own custom `jupyter-server` to which users can connect only through ssh. 6.1 will have support to connect to any JupyterHub running instance.
- I'm still unsure about the `common interface` part, but we'll see how that goes when Hendrick adds that support in the coming months.